cobc/flag.def

```diff
@@ -68,6 +68,11 @@ CB_FLAG_RQ (cb_max_errors, 1, "max-errors", 20, CB_FLAG_GETOPT_MAX_ERRORS,
 	  "                        * default: 20\n"
 	  "                        * if <number> is 0, there's no limit"))
 
+CB_FLAG_RQ (cb_source_encode, 1, "source-encode", 0, CB_FLAG_GETOPT_SOURCE_ENCODE, 
+	_("  -fsource-encode=[encoding]\tdefine the source file encoding\n"
+	  "                        * default: UTF-8"))
```

Comment:
I think it is reasonable to use the existing code as basis for the default - and that would be iso-8859-15; to override it people could use a BOM in the source file (pplex.l)